pylossless/dash/tests/test_topo_viz.py

```diff
@@ -52,7 +52,7 @@ def test_GridTopoPlot():
     offset = 2
     nb_topo = 4
     plot_data = topo_data.topo_values.iloc[::-1].iloc[offset : offset + nb_topo]
-    plot_data = list(plot_data.T.to_dict().values())
+    plot_data = plot_data.values.tolist()
```

Comment:
This is the problematic line. From brief inspection it seems like `GridTopoPlot` expects a list of `dict`'s, where for each `dict`, the keys are the ch_names and the value is a single `float`. The committed code from this passes a list of lists to `GridTopoPlot`, where each list just contains a series of `float`.

Just reverting this change fixes the test failure. But I assume you changed it for a reason, so I'm not sure if this will require a deeper inspection to the test to make sure it is testing for what we want it to?